### PR TITLE
Fix tcp-echo sample run panic.

### PR DESCRIPTION
**Pull request by yanggangtony**
Signed-off-by: yanggang <gang.yang@daocloud.io>

**Please provide a description of this PR:**
Build the sample ```samples/tcp-echo/src/main.go```, run it without parms, it panic without hint.
Fix it with parms valid and give hint for user..

/label release-notes-none

```release-note
None
```

### CODE REVIEW

**Comment by yanggangtony** [comment]
Hey, I want to know how to add a lable for the release-note-none?
Is that like k8s 's style?
```release-note
None
```

File: samples/tcp-echo/src/main.go [hunk 1]
```diff
@@ -25,6 +25,10 @@ import (
 
 // main serves as the program entry point
 func main() {
+	if len(os.Args) != 3 {
+		fmt.Println("you should run it like: 'main 9000,9001 hello' ")
```
Comment (ericvn) [inline samples/tcp-echo/src/main.go:29]: Maybe change to `main starting_port,ending_port prefix` to give the user a better idea on what the arguments represent.

Comment (yanggangtony) [reply samples/tcp-echo/src/main.go]: make sense ,  updated it.

File: samples/tcp-echo/src/main.go [hunk 2]
```diff
@@ -25,6 +25,10 @@ import (
 
 // main serves as the program entry point
 func main() {
+	if len(os.Args) != 3 {
+		fmt.Println("you should run it like: 'main starting_port,ending_port prefix' ")
```
Comment (ymesika) [inline samples/tcp-echo/src/main.go:29]: It can actually accept more than 2 ports so the description is a bit misleading.

Comment (ericvn) [reply samples/tcp-echo/src/main.go:29]: Maybe it should be port 1[,port 2]... ? I'm not sure of the best way to say a string of comma separated integers representing ports.

Comment (dhawton) [reply samples/tcp-echo/src/main.go:29]: `[PORTS...]` with a text description that ports are comma-delineated could be sufficient

Comment (yanggangtony) [reply samples/tcp-echo/src/main.go]: thanks all for the review.
Push again.
like 
```
if len(os.Args) != 3 {
		fmt.Println("you should run it like: 'main [PORTS...] prefix' , please notice PORTS are comma-delineated , like  port 1[,port 2]  ")
		os.Exit(1)
	}
```.

Would you think it is ok?

**Review by ericvn — APPROVED** [review]

**Pull request merged**